config/webpack.config.dev.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
         loader: 'json'
       },
       {
-        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)$/,
+        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)(\?.*)?$/,
```

Comment:
Yeah I'm not sure how else it's used in the wild, only encountered in with fonts. I wonder if other assets like CSS may need this.